v2/AdminCore.WebApp/Models/Holiday/CancelHolidayViewModel.cs

```diff
@@ -2,5 +2,6 @@
 {
   public class CancelHolidayViewModel
   {
```

Comment:
Is this something Mark approved? Personally i don't see why we bother with a view model for a single Id? I may be wrong though.

Reply:
If this is for approving holidays, it surely needs to know who approved it, when it was approved, is that taken care of by the JWT and the current date within the save method?

Reply:
a message might also be required